### ISSUE TITLE
Ensure helm chart is consistent with registry operator

### ISSUE THREAD

**Issue by michael-valdron** [devfile/api#1260]
## Which area/kind this issue is related to?

/area registry

## Issue Description

The setup for the [devfile registry helm chart](https://github.com/devfile/registry-support/tree/main/deploy/chart/devfile-registry) and [devfile registry operator](https://github.com/devfile/registry-operator) are currently inconsistent. We need to ensure that the registry helm chart setup is in sync with the setup of the registry operator in how they deploy the devfile registry before we can provide official releases #1009.

Related to #1193.

## Acceptance Criteria

- [ ] Ensure specifications under registry helm chart are consistent
  - Deployment specifications
  - Service specifications
  - Route specifications
  - Ingress specifications
  - Configmap specifications
  - Persistent Volume Claim (PVC) specifications
- [ ] Ensure helm chart parameters are consistent with `DevfileRegistry` CR, can provide the same setup, and use the same defaults
- [ ] Ensure [`Chart.yaml`](https://github.com/devfile/registry-support/blob/main/deploy/chart/devfile-registry/Chart.yaml) is consistent with the [registry operator cluster service version (CSV)](https://github.com/devfile/registry-operator/blob/main/config/manifests/bases/registry-operator.clusterserviceversion.yaml) file
- [ ] Ensure the helm chart supports the same [requirements](https://github.com/devfile/registry-operator/tree/main#requirements) as the registry operator
- [ ] Ensure hostname setup is consistent
- [ ] Ensure app name setup is consistent
- [ ] Update repository documentation to reflect the changes
- [ ] Update PR template to include acceptance criteria where if there is any changes to these items there should be a tied PR to update the [registry operator](https://github.com/devfile/registry-operator)

**Comment by michael-valdron** [comment]
Updated acceptance criteria to include additional PR acceptance criteria for the helm chart, setting for next refinement call.

**Comment by michael-valdron** [comment]
https://github.com/devfile/api/issues/1193#issuecomment-1741020515

**Comment by michael-valdron** [comment]
> https://github.com/devfile/api/issues/1193#issuecomment-1741200759

Due to the above change in direction this issue is no longer needed.

> Ensure the helm chart supports the same [requirements](https://github.com/devfile/registry-operator/tree/main#requirements) as the registry operator

This extra criteria has been included under #1193 to be part of the investigation.
